doc/nrf/known_issues.rst

```diff
@@ -231,36 +229,36 @@ Wait time required after a directed high duty cycle advertiser times out
 .. rst-class:: v1-1-0
 
 Assert risk after performing a DLE procedure
-  The :ref:`nrfxlib:softdevice_controller` 0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
+  The :ref:`nrfxlib:softdevice_controller` v0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
```

Comment:
Should we remove the SoftDevice Link Layer known issues from here?

Reply:
Done